library/cartridge_instance.py

```diff
@@ -14,21 +15,115 @@
 }
 
 
+def serialize_dict(d):
+    parts = []
+
+    for k, v in d.items():
+        value_str = None
+        if type(v) == str:
+            value_str = "'%s'" % v
+        elif type(v) == float or type(v) == int:
+            value_str = "%s" % v
+        elif type(v) == bool:
+            value_str = 'true' if v else 'false'
+        else:
+            raise CartridgeException(
+                cartridge_errcodes.BAD_VALUE_TYPE,
+                "Unknown value type: {}" % type(v)
+            )
+
+        parts.append("%s = %s" % (k, value_str))
+
+    return '{ %s }' % ', '.join(parts)
+
+
+def box_cfg_was_called(control_console):
+    return control_console.eval('''
+        return type(box.cfg) ~= 'function'
+    ''')
+
+
+def get_box_cfg(control_console):
+    return control_console.eval('''
+        return box.cfg or box.NULL
```

Comment:
Would you like to check that box.cfg is a table here?

Reply:
Yes, I agree. 

I'm trying to get rid of this check in every eval, see `box_cfg_was_called` function.